haiku-apps/beshare/beshare-2.31.recipe

```diff
@@ -49,19 +53,17 @@ BUILD_PREREQUIRES="
 	makefile_engine
 	"
 
-PATCHES="beshare-$portVersion.patchset"
-
 BUILD()
 {
 	ln -sf ../../sources-2/muscle muscle
```

Comment:
I don't know, however, if everyone would like such a change ;-)
(I guess some contributors prefer merging one-letter options. IMHO merging one-letter options is OK for commands typed in _interactive_ shells, and should not be used in scripts. Scripts should be readable even to someone who does not know anything about the ln command. But that's only my personal point of view, of course.)